sigs/sig-marketing/README.md

```diff
@@ -6,7 +6,7 @@ Argo Sig Marketing coordinates efforts between interested parties to promote, ad
 Sig marketing meetings are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
 
 ## Membership Requirements 
-- Members are nominated and approved by Argo Proj maintainers. 
+- Members are nominated and approved by other members and/or maintainers. 
```

Comment:
Should this say

"...other Marketing SIG Members and/or Argo Project Maintainers"

We should still be as specific as we can while maintaining flexibility

Reply:
Change made as suggested.